python/src/main/python/drivers/graphicsfuzz_piglit_converter.py

```diff
@@ -148,7 +152,9 @@ def make_test_header(shader_job_json_parsed: dict) -> str:
             uniform_name=uniform_name,
             args=' '.join([str(arg) for arg in value['args']])
         )
-    test_header += DRAW_COMMAND
+    test_header += CLEAR_COMMAND + '\n'
```

Comment:
I suppose this should be move down so it is under `if not nodraw`.

Reply:
Right - we wouldn't need to clear the screen if we're not going to draw to it in the first place.